### DISCUSSION TITLE
Splashscreen Warning Thing

### DISCUSSION THREAD

**Discussion by guyontheinternet2000** [TweakScale/TweakScale#321] [Support] [answered]
Kept getting the "Houston We Have A Problem" splashscreen but ignored it (Yes, I do go to the annual Idiot Convention.) Dunno what to do, halp

https://drive.google.com/file/d/1Ewqbc8EcdwTcl45arZXlsUDeA2I-gX6f/view?usp=drive_link

(Same guy who just posted on the KSP Forums, but my accounts new so the message needs to be aproved by a mod :p )

**Comment by Lisias** [comment, marked as answer]
Hi! I just answered it!

https://forum.kerbalspaceprogram.com/topic/179030-ksp-130-tweakscale-under-lisias-management-2475-2024-0213/page/160/#comment-4366891

TL;DR: 

* manually delete <KSP_ROOT>/GameData/NecroBones/ModularRocketSystems/Patches/TweakScale.cfg.
* Additionally, I strongly recommend you remove every instance of "MiniAVC.dll" from your rig. Your KSP.log is littered with exceptions due it.
* I suggest to install [TweakScale Companion the ÜberPaket](https://github.com/TweakScale/Companion/releases) (also available on CKAN)

**Comment by Lisias** [comment]
Assuming the ticket is concluded with success. Closing this.
